### PR TITLE
Updates for 1.7.0 with changes from main

### PR DESCRIPTION
**Pull request by spassarop**
- Updated pom.xml.
- Updated GitHub actions files.
- Updated test file.
- Fixed bug of tag smuggling inside style tags.

### CODE REVIEW

**Comment by spassarop** [comment]
@davewichers I updated pom.xml by hand comparing with main, could you check if it seems ok?

File: pom.xml [hunk 1]
```diff
@@ -41,9 +41,11 @@
     </scm>
 
     <properties>
+        <gpg.skip>true</gpg.skip><!-- by default skip gpg -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.outputTimestamp>2022-02-30T14:33:00Z</project.build.outputTimestamp>
-        <gpg.skip>true</gpg.skip><!-- by default skip gpg -->
```
Comment (davewichers) [inline pom.xml:46]: Why did you remove this?

Comment (spassarop) [reply pom.xml:46]: I did not, it is two lines above.

Comment (spassarop) [reply pom.xml:46]: In main is in that order, that is why I moved it up.

Comment (davewichers) [reply pom.xml:46]: OK. Looks good then.
